# BayerFormat Enum

string BAYER_GB = "BAYER_GB"
string BAYER_GR = "BAYER_GR"
string BAYER_BG = "BAYER_BG"
string BAYER_RG = "BAYER_RG"
